views/base.twig

```diff
@@ -1,45 +1,20 @@
+{% set title = 'App Name' %}
+
 <!DOCTYPE html>
 <html>
   <head>
-    <title>{{ title }}</title>
-    <link rel='stylesheet' href='/stylesheets/base.css'/>
+    <title>{% block title %}{{ title }}{% endblock %}</title>
     <link rel='stylesheet' href='/mdb/css/bootstrap.min.css'/>
     <link rel='stylesheet' href='/mdb/css/mdb.min.css'/>
     <link rel='stylesheet' href='https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css'/>
+    <link rel='stylesheet' href='/stylesheets/base.css'/>
```

Comment:
I figured out the weird font issue. Can't explain why, but it's because the base.css is loaded last. If we move it back under the <title>, we get our fancy font back :p